### PR TITLE
Create /admin section

### PR DESCRIPTION
**Pull request by marcusramberg**
This pull-request moves create post to /admin/create, adds a admin page
with a menu, and adds a settings editor.

A possible improvement would be to create the admin menu as a layout
that extends the main layout so the menu would be available for all the
settings pages.

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
 const router = new Router()
 const ctr = require('../controller')
+const adminCtr = require('../controller/admin')
 
-router.get(     '/',              async ctx => ctr.index(ctx))
-router.get(     '/create',        async ctx => ctr.create(ctx))
-router.post(    '/new',           async ctx => ctr.new(ctx))
+router.get(     '/',                async ctx => ctr.index(ctx))
+router.get(     '/admin',           async ctx => adminCtr.index(ctx))
+router.get(     '/admin/create',    async ctx => adminCtr.create(ctx))
```
Comment (arne) [inline routes/index.js]: I'd ideally like us to just use the ctr object here, and import adminCtr in the main controller.

Comment (arne) [reply routes/index.js]: If you agree I can fix that.

Comment (marcusramberg) [reply routes/index.js]: sure, go for it. Want me to resolve the conflicts first?

Comment (marcusramberg) [reply routes/index.js]: I just did it since it was trivial. Will await your routing change 👍

**Pull request merged**